bin/prepareapp.js

```diff
@@ -0,0 +1,267 @@
+#!/usr/bin/env node
+
```

Comment:
Where is this file being used? Is it needed to be committed? Lots of this code looks similar to what build_* scripts do.